liberty-maven-plugin/src/it/generate-features-it/src/test/java/net/wasdev/wlp/test/dev/it/GenerateFeaturesTest.java

```diff
@@ -95,4 +95,21 @@ public void customFeaturesTest() throws Exception {
         List<String> expectedFeatures = Arrays.asList("jaxrs-2.1");
         assertEquals(expectedFeatures, features);
     }
+
+    @Test
+    public void serverXmlCommentTest() throws Exception {
+        // initially the expected comment is not found in server.xml
+        File serverXmlFile = new File(tempProj, "src/main/liberty/config/server.xml");
+        assertFalse(verifyLogMessageExists(GenerateFeaturesMojo.FEATURES_FILE_MESSAGE, 10, serverXmlFile));
+        // also we wish to test behaviour when there is no <featureManager> element so test that
```

Comment:
Should we also have a test, or maybe an assertion in an earlier test for when there is a `<featureManager>` element in place?

Reply:
I'd go along with that. I was assuming we'd notice in manual tests but better to just auto test it.